### PR TITLE
IA Pages - add topics for the next languages

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
##### Description :
Adding the topics for the next programming languages

##### Reviewer notes :
screenshot-maria duckduckgo com 5001 2016-09-21 13-33-43

##### References issues / PRs:

#

##### Checklist :
- [ ] Back end tests (perl, scripts)
- [ ] Front end tests (js, integration)
- Browser verification
    - [ ] IE
    - [ ] Chrome
    - [x] Firefox
    - [ ] Safari
    - [ ] Opera 
- Mobile verification
    - [ ] iOS
    - [ ] Android

### CODE REVIEW

**Comment by tagawa** [comment]
Actually, I notice in the existing topics that we have a mixture of uppercase (e.g. JavaScript) and lowercase (e.g. "python") so maybe we should decide on a format first.

**Comment by MariagraziaAlastra** [comment]
@tagawa the ones with different casing have that because of how we show them on the front-end. We uppercase the first letter of each topic but we have to way to show JavaScript or CSS with the correct casing. Something to fix for the future :)

**Comment by tagawa** [comment]
@MariagraziaAlastra Thanks, sounds good. In the dropdown list for choosing topics python, etc. were lowercase, but I see in the user-facing pages the first letter is uppercase so all good.
Looks mergeable to me!

File: sql/1.187/languages.pl [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#
+use FindBin;
+use lib $FindBin::Dir . "/../../lib";
+use strict;
+use warnings;
+use DDGC;
+use Try::Tiny;
+
+my $d = DDGC->new;
+my @languages = [
```
Comment (jbarrett) [inline sql/1.187/languages.pl]:
```perl
my @languages = ( ... )
```

File: sql/1.187/languages.pl [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "ruby",
+    "Objective-C",
+    "R",
+    "scala",
+    "haskell"
+    ];
+
+for my $i (0 .. $#languages) {
+    my $temp_topic = $d->rs('Topic')->update_or_create({
+        name => $languages[0][$i]
```
Comment (jbarrett) [inline sql/1.187/languages.pl]:
```
for my $language (@languages) {
    $d->rs('Topic')->update_or_create({
        name => $language;
```

File: sql/1.187/languages.pl [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+use FindBin;
+use lib $FindBin::Dir . "/../../lib";
+use strict;
+use warnings;
+use DDGC;
+use Try::Tiny;
+
+my $d = DDGC->new;
+my @languages = [
+    "java",
```
Comment (jbarrett) [inline sql/1.187/languages.pl]: FYI, if you have single words to list like this, you can do:

```perl
qw/ java PHP HTML ... /
```

...to save typing all the quotes and commas

**Comment by MariagraziaAlastra** [comment]
@jbarrett All done, thanks!
schermata 2016-09-23 alle 07 30 08

**Comment by jbarrett** [comment]
:+1: LGTM, thanks!

**Pull request merged**